src/microbe_stage/systems/MicrobeEmissionSystem.cs

```diff
@@ -348,8 +340,16 @@ private void HandleSlimeSecretion(in Entity entity, ref MicrobeControl control,
                 jet.Active = false;
         }
 
-        control.QueuedSlimeSecretionTime -= delta;
-        if (control.QueuedSlimeSecretionTime < 0)
-            control.QueuedSlimeSecretionTime = 0;
+        DecreaseTimeCountdownValue(ref control.QueuedSlimeSecretionTime, delta);
+    }
+
+    private void DecreaseTimeCountdownValue(ref float countdownValue, float delta)
```

Comment:
I think this should have MethodImpl aggressive inlining set as this is basically de-inlining this method which could reduce performance quite a bit due to the extra method calls to do one small thing.